README.md

```diff
@@ -1,23 +1,91 @@
-```
-bot_token = "YOUR_TOKEN"
-```
 
-then
+## Contributing
+Our contributing workflow is as follows. This will setup a copy of the repository on your
+Github and push contributions without directly affecting the main repository.
 
+1. Fork a copy of the repository to your own Github by clicking on the top-right Fork button. 
+2. Clone the forked copy to your computer.
+3. Add a new Git remote pointing to the main repository with `git remote add upstream https://github.com/gweng88/zhenpai.git`.
+When you enter `git remote -v`, you should see two remotes, `origin` pointing to your forked repository
+and `upstream` pointing to the main repository. The output should look similar to below.
 ```
-pip3 install -r requirements.txt
-python3 bot.py
+origin	https://github.com/<your-github-username>/zhenpai.git (fetch)
+origin	https://github.com/<your-github-username>/zhenpai.git (push)
+upstream	https://github.com/gweng88/zhenpai.git (fetch)
+upstream	https://github.com/gweng88/zhenpai.git (push)
 ```
 
-### Stuff to do
+Whenever you want to create changes and submit them for review and merge, do the following workflow.
+
+1. Pull from upstream to update your local `main` branch: `git checkout main && git pull upstream`.
+You should never be directly committing to the `main` branch.
+2. Create a new dev branch with `git checkout -b <name-of-dev-branch>`.
+3. Make your changes.
+4. Push the changes to your forked repository by entering `git push origin <name-of-dev-branch>`.
+You should be able to see your changes on Github now.
+Run this command every time you want to keep adding changes to your branch after your first push.
+5. Make a pull request once you are satisfied with your changes and detail what you are adding/fixing.
+You can keep adding changes with `git push origin`, and your pull request will automatically be updated.
+6. A contributor will look over the pull request, leave feedback, and merge the commit back into `main`
+of the main repository.
+
+## Setup and Installation
+
+There are two ways of running the bot, on a local computer with a Python (>=3.7)
+installation (recommended for development) or inside a Docker container (recommended for
+deploying and hosting the bot).
+
+### Running the bot on a computer
+
+#### Prerequisites
+* Install Python 3.7 or newer
+* Install Git
+* Create a Discord application and retrieve its bot token at https://discord.com/developers/applications
+
+#### Steps
+1. Clone the repository.
+2. Open a terminal and change the working directory to the repository.
+3. (Optional) Set up a virtual environment in the repository.
+4. Install required Python packages in `requirements.txt` by entering `pip install -r requirements.txt`
+into the terminal.
+5. Create a `config.py` at the top-level directory of the repository with the contents
+of `config.example`. Replace the contents of `YOUR_TOKEN` with your bot token.
+6. Configure what cogs you want the bot to have by editing the `extensions` list in `bot.py`.
```

Comment:
```suggestion
6. **[Important]** Configure what cogs you want the bot to have by editing the `extensions` list in `bot.py`. You can do this to avoid setting up cogs that you don't care about.
```